tests/test_network_model_data.py

```
@@ -182,63 +120,84 @@
 ]
 
 
-# we need a small fixture to simulate the database here
+# ==============================================================================
+# Fixtures
+# ==============================================================================
 @pytest.fixture()
```

Ruff (0.14.1)

Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)